docs/src/pages/pt/getting-started.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Introdução
+---
+
+Astro é um moderno construtor de site estático. Aprenda mais sobre Astro através da [nossa página inicial](https://astro.build/) ou do [nosso post de lançamento](https://astro.build/blog/introducing-astro). Esta página é uma visão geral da documentação do Astro e de todos os recursos relacionados.
+
+Procurando por uma visão rápida do que é Astro? [Visite nossa página inicial.](https://astro.build)
+
+## Experimente Astro
+
+A maneira mais fácil de experimentar Astro é através da execução do comando `npm init astro` em um novo diretório na sua máquina. Nosso assistente para interface de linha de comandos (Wizard CLI) irá te ajudar a iniciar um novo projeto Astro.
+
+Para começar Astro em 5 etapas rápidas e fáceis, visite nosso [Guia de início rápido](quick-start).
+
+Como alternativa, leia nosso [Guia de Instalação](/installation) para uma guia de instalação completo do Astro.
+
+### Experimente Online
+
+Se você tem enteresse em testar Astro online no navegador, você pode usar um editor de código online. Experimente nosso template "Olá Mundo!" em [CodeSandbox](https://codesandbox.io/s/astro-template-hugb3).
+
+_Nota: algumas funcionalidades (ex: Fast Refresh) no momento estão limitadas no CodeSandbox._
+
+## Aprenda Astro
+
+Todos os tipos de pessoas vêm ao Astro com diferentes bagagems trazendo com eles diferentes estilos de aprendizagem. Se você prefere uma abordagem mais teórica ou prática, esperamos que você encontre esta seção útil.
+
+- Se você prefere **aprender fazendo**, comece com nossa [biblioteca de exemplos](https://github.com/snowpackjs/astro/tree/main/examples).
+- Se você prefere **aprender conceitos passo-a-passo**, comece com [conceitos básicos e guias](/core-concepts/project-structure).
+
+Como qualquer tecnologia desconhecida, Astro vem com uma ligeira curva de aprendizado. No entanto, com a prática e alguma paciência, sabemos que em pouco tempo você _vai_ pegar o jeito da coisa.
```

Comment:
A more natural way to say this would look something like
"Como qualquer tecnologia emergente, Astro vem com uma leve curva de aprendizado. No entanto, com prática e alguma paciência, sabemos que você _vai_ pegar o jeito da coisa."